packages/lib-classifier/src/components/Classifier/components/Feedback/FeedbackModal.spec.js

```diff
@@ -5,6 +5,11 @@ import sinon from 'sinon'
 import { Button } from 'grommet'
 import FeedbackModal from './FeedbackModal'
 
+const applicableRules = [
+  { id: '1' },
+  { id: '2' }
+]
+
```

Comment:
I could be wrong, but I thought the feedback viewer was set to only render a viewer if it was passed rules for the `graph2dRange` task. So I'm a bit confused as to how these mock rules will work.